sdk/android/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/table/query/QueryNodeODataWriter.java

```diff
@@ -47,7 +47,7 @@ private static String process(String s) {
     }
 
     private static String process(Date date) {
-        return "datetime'" + DateSerializer.serialize(date) + "'";
+        return "datetimeoffset'" + DateSerializer.serialize(date) + "'";
```

Comment:
Is this limited to the updatedAt column only?  We do not want all datetimes to go as datetimeoffset (That would break all Node backends)  This needs to be  al little more involved.